### PR TITLE
feat: Add Kube 1.31 support and drop 1.26

### PR DESCRIPTION
**Pull request by zeitlerc**
## Description

Test Kube 1.31 and drop 1.26

## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] Add changelog entry following the [contributing guide](../CONTRIBUTING.md#pull-requests)
- [ ] Documentation has been updated
- [x] This change requires changes in testing:
  - [x] unit tests
  - [x] E2E tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## v0.20.0 - 2024-10-17
+
+### 🚀 Enhancements
+- Add 1.31 support and drop 1.26 @zeitlerc [#574](https://github.com/newrelic/k8s-metadata-injection/pull/574)
+
```
Comment (csongnr) [inline CHANGELOG.md:14]:
```suggestion

### enhancement 
- Add 1.31 support and drop 1.26 @zeitlerc [#574](https://github.com/newrelic/k8s-metadata-injection/pull/574) 
```

the fun changelog w/ the emojis and stuff are generated. remove lines 10-13 and replace w/ the suggested change. sorry changelog sucks

Comment (csongnr) [reply CHANGELOG.md:14]: https://github.com/newrelic/release-toolkit/blob/main/README.md#validate-markdown

Comment (csongnr) [inline CHANGELOG.md:14]:
```suggestion
## v0.20.0 - 2024-10-17

###  enhancement
- Add 1.31 support and drop 1.26 @zeitlerc [#574](https://github.com/newrelic/k8s-metadata-injection/pull/574)

```

Comment (zeitlerc) [reply CHANGELOG.md:12]: I had no idea that was the process. Thanks

**Review by ramkrishankumarN — APPROVED** [review]

**Review by ramkrishankumarN — APPROVED** [review]

**Pull request merged**